lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -251,7 +251,7 @@ def initialize(model, current_user:, embed_links: false)
 
         linked_property :parent,
                         path: :work_package,
-                        title_getter: -> (*) { represented.parent.subject },
+                        title_getter: -> (*) { represented.parent.to_s },
```

Comment:
I originally wanted to move this out of the PR for this reason, but showing only the subject was so irritating that I decided against it.

Still, I agree with consistency and would argue we should change that around the API, since the mentioned format is more or less an agreed upon default in linking and relations.